testing/vidcutter/APKBUILD

```diff
@@ -7,7 +7,7 @@ pkgdesc="A modern, simple to use, constantly evolving and hella fast MEDIA CUTTE
 url="https://vidcutter.ozmartians.com"
 arch="all"
 license="GPL-3"
-depends="python3 ffmpeg mediainfo mpv py3-qt5"
+depends="python3 ffmpeg mediainfo mpv-dev py3-qt5"
```

Comment:
are the mpv development files needed for vidcutter runtime? wouldnt it be enought to add `mpv-dev` to `makedepends`?

Reply:
I guess not, i'll check.

Reply:
well the package is broken. it needs py3-opengl , py3-setuptools